node/src/components/consensus/highway_core/state.rs

```diff
@@ -62,6 +62,8 @@ pub(crate) enum VoteError {
     InstanceId,
     #[error("The signature is invalid.")]
     Signature,
+    #[error("The round length exponent has somehow changed within a round.")]
+    RoundExpLengthChangedWithinRound,
```

Comment:
Either `RoundExpChangedWithinRound` or `RoundLengthChangedWithinRound`.
The length of the round exponent would be the logarithm of the logarithm of the round length… :smile: 
(Or maybe you meant `RoundLengthExpChangedWithinRound`?)

Reply:
I meant `RoundLengthExpChangedWithinRound`.

Fixed in https://github.com/CasperLabs/casper-node/pull/479/commits/13a22ef3964dd9ad4f38e73682e5d6f46e56bd12